### PR TITLE
Pundit updates

### PR DESCRIPTION
**Pull request by TofuSupreme**
Hello everyone! Okay, so I've changed a few things around. Index page should be working now. I've added an H1 to it just for verification that it is working. (Also, I am going to delete my first PR and the other multiple branches which has the same thing)

### CODE REVIEW

File: app/policies/kondo_policy.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class KondoPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+    def index?
+      true
+  end
```
Comment (spioupiou) [inline app/policies/kondo_policy.rb:8]: Very minor but better correct the indentation?

File: app/policies/kondo_policy.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class KondoPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+
+    def index?
+      true
+    end
```
Comment (carlnoval) [inline app/policies/kondo_policy.rb:9]: We don't need these lines [as per the video at 57:00](https://kitt.lewagon.com/camps/718/lectures/05-Rails%2F07-Airbnb-Facebook-connect#video).

File: app/policies/kondo_policy.rb (hunk 1 above)
Comment (carlnoval) [reply app/policies/kondo_policy.rb:10]: I viewed the file and the indentations seem to be alright.
Screen Shot 2022-01-27 at 0 33 34

I was so paranoid I copy-pasted in vs code and the indentations seems right.

File: app/policies/kondo_policy.rb (hunk 2 above)
Comment (TofuSupreme) [reply app/policies/kondo_policy.rb]: Thanks for tip! I've updated the changes.

File: app/policies/kondo_policy.rb (hunk 1 above)
Comment (spioupiou) [reply app/policies/kondo_policy.rb:6]: Oh that’s strange! It must be my browser doing something weird! I resolve then :)

**Comment by spioupiou** [comment]
I tested the branch and confirmed I can access the homepage and index.
#show cannot be accessed yet as expected. 
I think we're good on that one, I'll merge!

Screen Shot 2022-01-27 at 9 00 07
Screen Shot 2022-01-27 at 9 03 19
Screen Shot 2022-01-27 at 9 03 24

**Pull request merged**